R/tar_renv.R

```diff
@@ -0,0 +1,56 @@
+#' @title Setup package dependencies for compatibility with `{renv}`
+#' @export
+#' @description Write package dependencies to a file named `_packages.R` in
```

Comment:
I think we need to emphasize that the lines in `_package.R` are all of the form `library(package)` and explicitly state that this helps `renv` pick them up. Should only be necessary if the user invokes `tar_option_set()` or `tar_target()` to declare packages rather than `library()` calls in `_targets.R`.